src/visitors/lessFileManager.js

```diff
@@ -0,0 +1,44 @@
+const Resolver = require('../Resolver');
```

Comment:
I don't think this file should go in `src/visitors`. Maybe just put it inside LESSAsset for now. Soon we will be splitting these assets out into their own packages, so multiple files per asset type will make more sense.

Reply:
Alright, just wanted to keep it clean, and wasn't sure where else to put it.